src/utils/utils.ts

```diff
@@ -581,4 +610,6 @@ export {
   isBufferSource,
   yieldMicro,
   setMaxListeners,
+  streamToAsyncGenerator,
+  asyncGeneratorToStream,
```

Comment:
I feel like these should really be in its own utility system.

Also don't we already have ixjs combinators available? Doesn't that already do this? Usually that library has these features.

Reply:
See the `ix` library 5.0.0. I feel like there would already be some combinator for async iterator to web streams.